dev-tools/mage/manifest/test_payload/manifest-8.14.0+build202406201002.json

```diff
@@ -0,0 +1,6167 @@
+{
+    "branch": "8.14",
+    "release_branch": "8.14",
+    "version": "8.14.0+build202406201002",
+    "build_id": "8.14.0+build202406201002",
+    "start_time": "Thu, 20 Jun 2024 10:02:13 UTC",
+    "end_time": "Thu, 20 Jun 2024 10:49:18 UTC",
+    "build_duration_seconds": 2825,
+    "manifest_version": "2.1.0",
+    "prefix": "",
+    "projects": {
+        "apm-server": {
+            "branch": "main",
```

Comment:
Not part of this PR, but seeing this as `main` is a bit weird since it should be `8.14.